src/api/helpers/decrypt/generate-decipher/index.test.ts

```diff
@@ -0,0 +1,19 @@
+import crypto from 'crypto';
+import generateDecipher from '.';
+import generateKey from '../../encrypt/generate-key';
+import { FINANCIAL_DETAILS } from '../../../constants';
+import { mockIV } from '../../../test-mocks';
+
+const { ENCRYPTION_METHOD } = FINANCIAL_DETAILS.ENCRYPTION.CIPHER;
+
+describe('api/helpers/decrypt/generate-decipher', () => {
+  const key = generateKey();
+  const iv = mockIV;
+  const decipher = crypto.createDecipheriv(ENCRYPTION_METHOD, key, iv);
+
+  it('should return a valid key', () => {
```

Comment:
Assertion for following please with the arguments combinations:
* An empt string arguments.
* Not a string type arguments.

Reply:
Same as above